src/assets/src/components/queueManager.tsx

```diff
@@ -212,7 +212,13 @@ function QueueManager(props: QueueManagerProps) {
             </Table>
         )
         : (
-            <Alert variant="dark">No meetings in queue.</Alert>
+            <>
+            <hr/>
+            <h5>No meetings in Queue</h5>
+            <p>
+                Did you know? You can get notified by SMS (text) message when someone joins your empty queue by adding your cell phone number and enabling host notifications in your <Link to="/preferences">User Preferences</Link>. 
+            </p>
```

Comment:
I know @teshsopher suggested removing the Alert formatting, but semantically this feels to me like an alert; that would also be consistent with how we are doing it for the attendee. I also find the use of `<hr/>` here to be a little different from what we've been doing elsewhere, where it's kind of a footer separator.

Reply:
If we want to be more consistent, using the Alert formatting on the host side is fine for this release. However, I think one of my next goals is to reduce the number of Alert-style boxes, particularly on the attendee side, to improve overall clarity of ROHQ.

Reply:
I will defer to @teshsopher. I guess, just because this is a conditionally rendered block, the Alert made more sense to me. I can understand not wanting to overuse that info presentation method.